usr/share/rear/layout/prepare/GNU/Linux/130_include_mount_subvolumes_code.sh

```diff
@@ -1,11 +1,14 @@
 
```

Comment:
On this occation, the file could be renamed to something like `130_include_btrfs_subvolumes_SLES_code.sh`.

Reply:
Done via
https://github.com/rear/rear/pull/2080/commits/cd2bd2e60a6824b2244ef91abb34f8d239dcb500